libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -303,7 +305,11 @@ proc handleControl(g: GossipSub, peer: PubSubPeer, control: ControlMessage) =
     trace "sending control message", msg = shortLog(respControl), peer
     g.send(
       peer,
-      RPCMsg(control: some(respControl), messages: messages))
+      RPCMsg(control: some(respControl)), true)
+    # iwant replies have lower priority
```

Comment:
there might not be any messages or controls to send, ie either of these may be empty.

Reply:
Won't line 284 handle that?

Reply:
284 is an `or` condition

Reply:
yeah, seems to be correct now.